test/test-beebop.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# Copyright 2018-2024 John Lees and Nick Croucher
+
+"""Tests for PopPUNK --update-db order"""
+
+import subprocess
+import os, sys
+import sys
+import shutil
```

Comment:
To get this to run in the CI you can add [here](https://github.com/bacpop/PopPUNK/blob/master/.github/workflows/azure_ci.yml#L39) or add `subprocess.run('test-beebop.py')` to `test/run_test.py`

But, I would actually suggest just copying lines 16-19 below into the end of `test/run_test.py`

Reply:
Great, done in d2421f1.